content/en/docs/dependencies/merge-humanitec-deployments.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Merge Deployment Deltas with score-humanitec"
+linkTitle: "Merge Deployment Deltas"
+weight: 6
+draft: true
+description: >
+  Describes how to set merge Deployment Deltas.
+---
+
+As a developer, you may need to deploy multiple Workloads for your application simultaneously. Deploying multiple Workloads can be a complex process, as it requires you to manage the dependencies between the different Workloads. To simplify this process, you can use the --delta flag to deploy multiple Workloads as a single release.
+
+In this tutorial, we will show you how to merge two workload deployments together using the --delta flag. We will use the Score implementation as an example to demonstrate the process, and you can follow along with the steps to merge your workload deployments seamlessly.
+
+Before we dive into the tutorial, it's important to understand what a deployment delta is. A deployment delta is a representation of the changes made to your application deployment. It's a small, lightweight file that captures the changes between two deployments. By using deployment deltas, you can apply incremental changes to your application deployments, which makes it easier to manage and track changes.
```

Comment:
Would it make sense to link to the Humanitec docs here? Not sure if that's something we do in other parts of the Score docs

Reply:
Makes sense!